packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -1218,6 +1218,32 @@ void main() {
 
     await gesture.up();
   });
+
+  testWidgets('activeTrackRadius is taken into account when painting the border of the active track', (WidgetTester tester) async {
```

Comment:
This test still passes without the fix, can you make sure the fix is tested?

Reply:
The `additionalActiveTrackHeight` is not exposed anywhere to the user, so the only way to test this without introducing new code is to extend from `RoundedRectSliderTrackShape`, override the `paint` function, and set the default value of `additionalActiveTrackHeight` to something other than 2.0. Is there a faster approach?